.eslintrc.js

```diff
@@ -0,0 +1 @@
+module.exports = require('eslint-config-wolox-node');
```

Comment:
Take a look at this line, where ESLint is configured with a set of custom rules. Usually you should add here any change to the slint rules.

Reply:
(There is nothing to do, just a comment)